src/test/fuzz/cluster_linearize.cpp

```diff
@@ -559,6 +559,10 @@ FUZZ_TARGET(clusterlin_simple_finder)
                 auto exhaustive = exh_finder.FindCandidateSet();
                 assert(exhaustive.feerate == found.feerate);
             }
+
+            // Compare with a topological set read from the fuzz input.
+            auto read_topo = ReadTopologicalSet(depgraph, todo, reader);
```

Comment:
We can cache the subset and then reuse it here and when finding topological subset to be deleted.